drv/fpga-app-api/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! API crate for shared pieces of logic used in FPGA applications.
+
+#![no_std]
+
+pub mod power_rail;
```

Comment:
Why put this in a separate module, versus the existing `fpga-api`?  Are there more pieces of `fpga-api` that should move here eventually?

Reply:
The intend is that `fpga-api` deals with FPGA devices (reset, loading bitstreams etc.) and `fpga-app-api` would be APIs for logic contained within bitstreams and running on those FPGAs. An alternative name for this crate could be `fpga-design-api`. The point is that there is RTL we're starting to share between multiple tasks and I figured I'd start pulling some of that into it's own crate. Happy to hear suggestions.

Reply:
Maybe `fpga-user-api`, since we use "user design" to refer to the running code?